presentation/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,10 @@
 <resources>
-
     <string name="app_name">MU8</string>
+
+    <string name="error_internet_message">Internet Error, please check your connection</string>
     <string name="version_name">v %1$s</string>
 
+    <string name="toolbar_title_comics">Comics</string>
     <string name="characters_name">Characters</string>
-
+    <string name="category_section_comics">Comics</string>
```

Comment:
How is this different from the `toolbar_title_comics` with the same value? Let's use the same or rename the resource for something more general that we can use anywhere.